Dockerfile

```diff
@@ -115,12 +115,6 @@ COPY . .
 COPY --from=argocd-ui /src/dist/app /go/src/github.com/argoproj/argo-cd/ui/dist/app
 RUN make argocd-all
 
-ARG BUILD_ALL_CLIS=true
```

Comment:
Sorry, I did not catch it earlier. The Makefile has `release-cli` target to expects these binaries in image: 

https://github.com/argoproj/argo-cd/blob/8bbf8887d25c1c200e88ff3e8ec228d4cd637599/Makefile#L215-L219

Binaries are uploaded to github by by release workflow

https://github.com/argoproj/argo-cd/blob/8bbf8887d25c1c200e88ff3e8ec228d4cd637599/.github/workflows/release.yaml#L191


Can you move  `make BIN_NAME=argocd-darwin-amd64 GOOS=darwin argocd-all && make BIN_NAME=argocd-windows-amd64.exe GOOS=windows argocd-all ` to `release-cli` target please?

Reply:
Good catch. I changed this to an env var and it now defaults to `false` except for `make release-cli` in the release workflow.